File: Day-14/Pattern16.js
function PatternSixteen(n) {
  let i = 1;
  let ch = 65;
  while (i <= n) {
    let j = 0;
    let str = "";
    while (j < i) {
      str += String.fromCharCode(ch);
      j++;
      ch++;
    }
    i++;
    console.log(str);
  }
}

PatternSixteen(4);

File: Day-14/Pattern16.py
n = int(input())
def PatternSixteen(n):
    i = 1
    ch = 65
    while(i <= n):
        j = 1
        while(j <= i):
            print(chr(ch), end='')
            ch += 1
            j += 1
        i += 1
        print(end='\n')

PatternSixteen(n)